.gitmodules

```diff
@@ -164,3 +164,9 @@
 [submodule "vendor/src/django-sundial"]
 	path = vendor/src/django-sundial
 	url = https://github.com/charettes/django-sundial
+[submodule "vendor/src/responses"]
+	path = vendor/src/responses
+	url = https://github.com/getsentry/responses.git
+[submodule "vendor/src/cookies"]
```

Comment:
This project lacks a ``.gitignore`` line for ``*.pyc``.  Can you add ``ignore = untracked`` so that ``cookies.pyc`` won't show up in ``git status``?